### PR TITLE
EL-4146: Color Service Aliases

### PR DESCRIPTION
**Pull request by JonathanToddMF**
#### Checklist

* [x] The contents of this PR are mine to submit. No third party code or other material is being committed.
* [x] New third party dependencies (if any) are linked via `package.json`. Third party dependencies are licensed under one of the following: Apache, MIT, BSD, Mozilla Public License, Eclipse Public License, or Oracle Binary Code License.
* [x] The code in this PR does not contain export-restricted encryption algorithms.

#### Ticket / Issue
https://portal.digitalsafe.net/browse/EL-4146

#### Description of Proposed Changes
1. Updated micro-focus-color-set 
2. Adding tests for color service

#### Documentation CI URL

https://pages.github.houston.softwaregrp.net/sepg-docs-qa/UXAspects_CI_UXAspects_EL-4146-color-service-aliases

#### Downstream Build(s)

[us-aspects-micro-focus](https://jenkins.swinfra.net/job/SEPG/job/caf/job/caf~ux-aspects-micro-focus~EL-4146-color-service-aliases~CI/)

### CODE REVIEW

**Comment by cmallon367** [comment]
You forgot to fill in the downstream build part of the PR

**Comment by donna-marijne** [comment]
I strongly suggest using the [UX Aspects userscripts](https://github.houston.softwaregrp.net/caf/ux-aspects-team/tree/master/userscripts) - save yourself a lot of typing when making a pull request.

File: src/services/color/color.service.spec.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+describe('Color Service - Micro Focus Color Set', () => {
+
+  beforeEach(() => TestBed.configureTestingModule({
+    imports: [ColorServiceModule.forRoot(colorSets.microFocus)],
+    providers: [ColorService]
+  }));
+
+  it('should return the correct rgb values for aliases', () => {
+    const service: ColorService = TestBed.inject(ColorService);
+    service.getColorSet();
```
Comment (donna-marijne) [inline src/services/color/color.service.spec.ts:16]: I don't think this line is needed.

**Review by donna-marijne — CHANGES_REQUESTED** [review]
This looks fine, I think there's some redundant code though.

File: src/services/color/color-sets/micro-focus-color-set.ts [hunk 2]
```diff
@@ -64,8 +64,10 @@ export const MICRO_FOCUS_COLOR_SET = {
     'chart4': '#ffce00',
     'chart5': '#eb23c2',
     'chart6': '#ba47e2',
+    'info': '00abf3',
```
Comment (donna-marijne) [inline src/services/color/color-sets/micro-focus-color-set.ts:67]:
```suggestion
    'info': '#00abf3',
```
Maybe should fix `critical` there as well.

**Review by donna-marijne — CHANGES_REQUESTED** [review]

**Review by donna-marijne — APPROVED** [review]
Thank you, it looks good.

**Pull request merged**